models/settings_helpers.go

```diff
@@ -129,6 +134,14 @@ func UpdateSettings(q reform.DBTX, params *ChangeSettingsParams) (*Settings, err
 		settings.SaaS.STTEnabled = true
 	}
 
+	if params.SessionID != "" {
+		settings.SaaS.SessionID = params.SessionID
+	}
+
+	if params.Email != "" {
+		settings.SaaS.Email = params.Email
+	}
```

Comment:
How that code will handle sign-out? I guess we will have to add boolean flag like `RemoveAlertManagerURL`, but can we do something better?

Reply:
We haven't signout mechanics for now. It requires also session termination before deleting sessionID in managed.
There should be another ticket created for that, I think.

Boolean flag looks ok, at least I don't see something more simple for now. It could be "LogOut bool", and when it's true we remove `Email` and `SessionID`.

Reply:
Add such filed now, or leave it to another ticket?

Reply:
The intention was not do anything that will make future sign-out functionality worse / harder.
If boolean flag is the best we can think of, so be it

Reply:
Done